src/components/Card.js

```diff
@@ -25,41 +53,87 @@ export default class Card {
       .addEventListener('click', ()=> {
         this._handlePreviewDisplay();
       });
+  }*/
+
+  _showCardLikes() {
+    this._cardLikeCounter.textContent = this._likes.length;
+    if (this.cardIsLiked) {
+      this._cardLikeButton.classList.add("cards__like-button_active");
+    } else {
+      this._cardLikeButton.classList.remove("cards__like-button_active");
+    }
+  }
+
+  _renderLikes() {
+    if (this.isLiked) {
+      this._cardLikeButton.classList.add("cards__like-button_active");
+    } else {
+      this._cardLikeButton.classList.remove("cards__like-button_active");
+    }
+  }
+
+  updateLikes(isLiked) {
+    this.isLiked = isLiked;
+    this._renderLikes();
   }
 
   _handleLikeButton() {
     this._cardElement
-      .querySelector('.element__like-button')
-      .classList.toggle('element__like-button_active');
+      .querySelector(".element__like-button")
+      .classList.toggle("element__like-button_active");
   }
 
   _handleDeleteButton() {
     this._cardElement.remove();
     this._cardElement = null;
   }
 
-  _handlePreviewDisplay() {
-    this._handleCardClick(this._name, this._url);
+handleDeleteCard() {
```

Comment:
Needs correction: you don't need to have this method, you have another one on line 96